### PR TITLE
Thomasdeleval patch warehouses

### PR DESCRIPTION
**Pull request by thomasdeleval**
changed the warehouses docs, added inventory adjustments as well

### CODE REVIEW

File: content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+.. image:: count_products/request-count.png
+   :align: center
+   :alt: history of moves
+     
+.. tip::
+    The report offers a lot of filters that you can use to make your selection
+    
+By default, after an adjustment is applied, the scheduled date for the next 
+count is the 31th of december of the current year. This is a setting that can
```
Comment (jirocha) [inline content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:88]: Only changes here would be "31th" to "31st"

File: content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    consistant with the difference. When applying such line, Odoo will ask if you
+    want to keep the difference and the count.
+   
+Plan counts
+===========
+
+On each line there is:
+- *Scheduled Date*: it's the date at which a count should be made.
+- *User*: it's the person in charge of the count.
+- *Accounting Date*: this is the dat at which the adjustments will be accounted. The
```
Comment (jirocha) [inline content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:65]: "dat" to "date"

File: content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+
+.. image:: count_products/apply-all.png
+   :align: center
+   :alt: apply all wizzard allows to change reason 
+
+Count products
+==============
+
+Counting products is a recurent activity in a warehouse. It often happens that theoric
```
Comment (jirocha) [inline content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst:40]: `Counting products is a recurent activity` to `Counting products is a recurring activity`

**Review by jirocha — CHANGES_REQUESTED** [review]
Just a few small edits then should be fine. Thank you :)

File: content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst (hunk 1 above)
Comment (thomasdeleval) [reply content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst]: done

File: content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst (hunk 2 above)
Comment (thomasdeleval) [reply content/applications/inventory_and_mrp/inventory/management/inventory_adjustments/count_products.rst]: done

**Review by jirocha — APPROVED** [review]

**Comment by jcs-odoo** [comment]
@StraubCreative moved this PR's content to another PR #2556 .
I close this one to avoid duplicates and to clean the list.

**Pull request closed without merging**